.github/workflows/Create-Release-Tags.yml

```
@@ -0,0 +1,26 @@
+name: Create release tags
+on:
+  workflow_dispatch: # Allow manual trigger
+    inputs:
+      agent-ref:
+        description: 'The ref (release tag) to create init containers for'
+        required: false
+        default: 'main'
+        type: string
+jobs:
+  create_release_tags:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # pin@v4
+        with:
+          ref: ${{ inputs.agent-ref }}
+      - name: Create release tags for K8s Init Container
+        run: |
+          RELEASE_TITLE="New Relic Java Agent ${GITHUB_REF}.0"
+          RELEASE_TAG="${GITHUB_REF}.0_java"
+          RELEASE_NOTES="Automated release for [Java Agent ${GITHUB_REF}](https://github.com/newrelic/newrelic-java-agent/releases/tag/${GITHUB_REF})"
```

As far as I know, `GITHUB_REF` doesn't exist for manually triggered builds. Replace with `inputs.agent-version`. 
```suggestion
          RELEASE_TITLE="New Relic Java Agent ${inputs.agent-version}.0"
          RELEASE_TAG="${inputs.agent-version}.0_java"
          RELEASE_NOTES="Automated release for [Java Agent ${inputs.agent-version}](https://github.com/newrelic/newrelic-java-agent/releases/tag/${inputs.agent-version})"
```